content/en/docs/languages/java/api.md

```diff
@@ -1,8 +1,1691 @@
 ---
-title: API reference
-linkTitle: API
-redirect: https://javadoc.io/doc/io.opentelemetry
-manualLinkTarget: _blank
-_build: { render: link }
-weight: 210
```

Comment:
This needs to stay somewhere, we can think about renaming them to "api-reference.md" but would need to apply that to all languages. 

Why not name the new page "record-telemetry"?

cc @chalin as this is related to our other dicussions around API References/API Docs

Reply:
Let's keep the `api` path across all languages.
If you'd like a Java-specific alias, add, e.g.:

```yaml
aliases: [javadoc]
```

Reply:
Ah I see. I was wondering why the build was broken and it sounds like the `/api/` path is a special case. 

Any recommendations on what I name the file which documents all the components of the API? `telemetry-api.md`, `api-doc.md`, `api-ref.md`, `java-api.md`, something else?

Reply:
What about `/api-components` ?